composer.json

```diff
@@ -19,7 +19,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6",
```

Comment:
may I ask why the lib requires 5.6? could see a 5.6+ only feature on the first glance